awx/main/tests/unit/tasks/test_jobs.py

```
@@ -136,9 +141,10 @@ def test_pre_post_run_hook_facts(mock_create_partition, mock_facts_settings, pri
 
 
 @mock.patch('awx.main.tasks.facts.bulk_update_sorted_by_id')
+@mock.patch('awx.main.tasks.jobs.flag_enabled', return_value=False)
 @mock.patch('awx.main.tasks.facts.settings')
 @mock.patch('awx.main.tasks.jobs.create_partition', return_value=True)
-def test_pre_post_run_hook_facts_deleted_sliced(mock_create_partition, mock_facts_settings, private_data_dir, execution_environment):
+def test_pre_post_run_hook_facts_deleted_sliced(mock_create_partition, mock_facts_settings, mock_flag_enabled, private_data_dir, execution_environment):
```

Ruff (0.15.2)

[warning] Unused function argument: `mock_create_partition`

(ARG001)